docs/_docs/framework/capabilities/manage/finops-assessment.md

```diff
@@ -0,0 +1,35 @@
+<a name="definition"></a>
+**FinOps Assessment refers to assessing FinOps practice and measure its effectiveness, align activities with organizational goals, and identify areas for improvement, with the aim of becoming more mature.**
+{: .fs-6 .fw-300 }
+
+FinOps Capabilities can be evaluated at any point in an organization’s FinOps journey.
+The objective of the FinOps assessment is to capture the maturity of a set of activities, define the objectives according to the organization, and outline the path to achieve that goal. Setting KPIs will help the FinOps team identify which objectives are being met and which areas need more attention to increase maturity.
+When conducting an assessment, organizations can evaluate their current capabilities and reassess the capabilities that will have the most impact on maximizing cloud value. These assessments are carried out by a target group consisting of a group composed of different personas that bring different points of view on what is essential. As organizations gain maturity, they will implement more capabilities.
+
+<br>
+
+
+## ✋ Before you begin
+
+For organizations that are just starting out, they can conduct the Microsoft FinOps Review. 
```

Comment:
Should we include a direct link to the FinOps Review assessment? 

[Microsoft FinOps Review](https://aka.ms/finops/review)

Reply:
```suggestion
For organizations that are just starting out, they can conduct the [Microsoft FinOps Review](https://aka.ms/finops/review) 
```